artpark_robot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100

  # Effort Controllers ---------------------------------------
  trash_holder_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: trash_holder_joint
    # pid: {p: 100.0, i: 0.01, d: 10.0}

  arm_base_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: arm_base_joint
    # pid: {p: 100.0, i: 0.01, d: 10.0}

  body_joint_effort_controller:
    type: effort_controllers/JointPositionController
    joint: body_joint
    pid: {p: 5.0, i: 0.1, d: 0.15}

  front_left_wheel_caster_joint_effort_controller:
    type: effort_controllers/JointPositionController
    joint: front_left_caster_joint
    pid: {p: 15.0, i: 0.1, d: 0.15}
  
  front_right_wheel_caster_joint_effort_controller:
    type: effort_controllers/JointPositionController
    joint: front_right_caster_joint
    pid: {p: 15.0, i: 0.1, d: 0.15}
  
  rear_left_wheel_caster_joint_effort_controller:
    type: effort_controllers/JointPositionController
    joint: rear_left_caster_joint
    pid: {p: 15.0, i: 0.1, d: 0.15}
  
  rear_right_wheel_caster_joint_effort_controller:
    type: effort_controllers/JointPositionController
    joint: rear_right_caster_joint
    pid: {p: 15.0, i: 0.1, d: 0.15}

  # body_position_controller:
  #   type: effort_controllers/JointPositionController
  #   joint: body_joint
  #   pid: {p: 5.0, i: 0.0, d: 0.15}
  
  #front_right_wheel_joint_effort_controller:
   # type: effort_controllers/JointEffortController
    #joint: front_right_wheel_joint
    # pid: {p: 100.0, i: 0.01, d: 10.0}
  
  #rear_left_wheel_joint_effort_controller:
   # type: effort_controllers/JointEffortController
    #joint: rear_left_wheel_joint
    # pid: {p: 100.0, i: 0.01, d: 10.0}

  #rear_right_wheel_joint_effort_controller:
   # type: effort_controllers/JointEffortController
    #joint: rear_right_wheel_joint
    # pid: {p: 100.0, i: 0.01, d: 10.0}
  